hubot-rocketchat/docker-compose.yml

```diff
@@ -9,8 +9,14 @@ services:
     - LISTEN_ON_ALL_PUBLIC=true
     - ROCKETCHAT_USER=meeseeks
     - ROCKETCHAT_PASSWORD=pass
+    - TENOR_API_KEY=
```

Comment:
Ok.. the `compose` has got a thing called [.env-files](https://docs.docker.com/compose/environment-variables/#set-environment-variables-in-containers), which you can exclude from the VCS and keep them locally :)